### DISCUSSION TITLE
I have : error SS0006: The graphQLData cannot be empty. (Parameter 'graphQLData')

### DISCUSSION THREAD

**Discussion by bsdnetpl** [ChilliCream/hotchocolate#5808] [Q&A]
Hello, I'm trying to write a tutorial based on: (https://chillicream.com/docs/strawberryshake/v13/get-started) to make a connection between blazor and graphql but I get this error: CSC : error SS0006: The graphQLData cannot be empty. (Parameter 'graphQLData') What am I doing wrong?
(My repository: https://github.com/bsdnetpl/Demo/tree/master/Demo)

**Comment by PascalSenn** [comment]
It seems like you GraphQL file is empty:
https://github.com/bsdnetpl/Demo/blob/master/Demo/GetSessions.graphql

**Reply by bsdnetpl** [reply]
Everything is fine in my project:

query GetAssets {
   assets {
     nodes {
       name
       price {
         lastPrice
       }
     }
   }
}
  Probably some error in commit. I'll make a good commit on Monday at work.
